lib/Parse/Lexer.cpp

```diff
@@ -1244,6 +1244,9 @@ static const char *skipToEndOfInterpolatedExpression(const char *CurPtr,
                                                      DiagnosticEngine *Diags,
                                                      bool MultilineString) {
   llvm::SmallVector<char, 4> OpenDelimiters;
+  llvm::SmallVector<bool, 3> AllowNewline;
```

Comment:
Very minor nitpick: why 3? Might as well go with 4 or 8, as the small vector will be pointer-size aligned. This doesn't block this PR, though.